src/tests/test_magic.py

```diff
@@ -2248,3 +2248,46 @@ def test_warn_when_using_snippets_in_non_select_command(
 )
 def test_get_query_type(query, query_type):
     assert get_query_type(query) == query_type
+
+
+@pytest.mark.parametrize(
+    "query, expected",
+    [
+        (
+            """%%sql
+SELECT 1""",
+            1,
+        ),
+        (
+            """%%sql
```

Comment:
Please add tests for comments at the start/middle of query and then again at the end after semicolon

Reply:
Added [here](https://github.com/ploomber/jupysql/pull/939/files#diff-d327143929530bcece2cae65d59465708cdf87f499fd83b4bd939c332150f713R2289-R2329)